app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,128 @@
+
+<li>write creatively to express their own ideas and opinions</li>
+</ul>
+
+You can explore:
+<ul>
+  <li><a href="https://www.thenational.academy/teachers/programmes/french-secondary-ks3-l/units">examples of French lessons</a></li>
+  <li><a href="https://www.thenational.academy/teachers/programmes/german-secondary-ks3-l/units">examples of German lessons</a></li>
+  <li><a href="https://www.thenational.academy/teachers/programmes/spanish-secondary-ks3-l/units">examples of Spanish lessons</a></li>
+  </ul>
+    </div>
+</section>
+  <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "You build bridges connecting individuals across cultures, fostering understanding, collaboration, and personal growth.",
+    name: "Languages teacher",
+    large: true
+  ) %>
+</section>
+  <section class="col col-space-m col-720">
+     <h2>Check your qualifications</h2>
+<p>To train to teach in primary and secondary schools in England, you’ll need: </p>
+<ul>
+<li>GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary) </li>
+<li>a bachelor’s degree in any subject </li>
+</ul>
+<p>Modern languages teacher training courses have had successful applications from candidates with a range of degrees, including international relations, psychology, socialogy and journalism.</p>
+<p><a href="/non-uk-teachers/train-to-teach-in-england-as-an-international-student">Non-UK citizens can find out more about training to teach in England</a>.</p>
+
+ <p>Your training provider may need you to brush up on your computing knowledge before you become a modern languages teacher. You can do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE)</a> course before you start your teacher training to top up your subject knowledge. </p>
+
+<p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out more about the qualifications needed to teach</a>.</p>
+    <h2> Fund your teaching training </h2>
+
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free scholarships of £27,000 are available for eligible trainee French, German, and Spanish languages teachers</a>. </p>
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £25,000 are available for all eligible trainee language teachers, including ancient languages</a>. </p>
+
+  <p>You can receive this alongside a tuition fee and maintenance loan. </p>
+
+  <p>You may also be able to get <a href="/funding-and-support">extra funding support</a> if you are: </p>
+  <ul>
+<li>a parent or carer </li>
+<li>disabled </li>
+</ul>
+<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding. </p>
+<p>You can <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">find out more about fees and financial support available for non-UK citizens</a>.</p>
```

Comment:
@caitlin-reidy don't think it needs You can. Could just start the sentence with the link as Find out more...